app/scripts/modules/core/src/orchestratedItem/orchestratedItem.transformer.ts

```diff
@@ -86,7 +86,7 @@ export class OrchestratedItemTransformer {
       status: {
         // Returns either SUCCEEDED, RUNNING, FAILED, CANCELED, or NOT_STARTED
         get: (): string => this.normalizeStatus(item),
-        set: status => {
+        set: (status) => {
```

Comment:
i guess webstorm did this...? i can undo if needed

Reply:
prettier does what prettier wants

... but this seems to be only contain formatting changes.  might as well revert this file